dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/GeneratorPVAlphaSum.mo

```diff
@@ -0,0 +1,73 @@
+within Dynawo.Electrical.Machines;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model GeneratorPVAlphaSum "Model for generator PV with AlphaSum."
+
+  extends BaseClasses.BaseGeneratorSimplified;
+  extends AdditionalIcons.Machine;
+
+  public
+
+    parameter Types.PerUnit KGover = 1 "Mechanical power sensitivity to frequency";
```

Comment:
Do you need to keep this parameter?

Reply:
I have two solutions :
- I take AlphaRaw=PNom or PMax, etc. from the IIDM and I need this KGover
- We find a way to put it in the AlphaRaw beforehand and we don't need it
What do you think ?

Reply:
My point is: this KGover is by default to 1 and I don't see it in the parameters files of your tests. Could we just have:
PGenRawPu = PGen0Pu + (Alpha  / alphaSum.value) * N.value;
alpha= if (N.value > 0 and PGenRawPu >= PMaxAuxPu) then 0 else if (N.value < 0 and PGenRawPu <= PMinAuxPu) then 0 else Alpha;
?

Reply:
Yes we could and this is how it was done at first. In fact for these small cases it is not used.
When I tested the Pt-Fige, I also took into account the KGover used in Dynawo voltage stability as another parameter. I we manage to have a proper AlphaRaw parameter that takes it into account (when KGover = 0 for example, AlphaRaw must be 0 too), we don't need it. Maybe we can construct the AlphaRaw from the PMax/PNom/P0 and the droop/KGover before and therefore we don't need a KGover parameter.

Reply:
Thanks for you answer, and how do you set the value for AlphaRaw?

Reply:
For the tests that I have done, I use the same PMax as you do in Dynawo voltage stability (the one for the governor I guess). Hence the need for a KGover in the model, for the cases where the generator doesn't participate or does but with a different factor (not necessary 25 for everyone).
So do you prefer that we keep it or integrate it in the AlphaRaw when setting it ?
Thanks